### ISSUE TITLE
Part07 Error#betterCallSaulForBogusMono() better with onErrorReturn

### ISSUE THREAD

**Issue by letsky** [reactor/lite-rx-api-hands-on#68]
Hi, I am learning reactor on tech.io. In the first test case, it prompts me to use `onErrorReturn()`, but `solution` branch uses `onErrorResume()`. Both methods are correct, i think it is better to use `onErrorReturn()` in this case.

In `solution` branch
```java
Mono<User> betterCallSaulForBogusMono(Mono<User> mono) {
    return mono.onErrorResume(e -> Mono.just(User.SAUL)); // TO BE REMOVED
}
```
My solution
```java
Mono<User> betterCallSaulForBogusMono(Mono<User> mono) {
    return mono.onErrorReturn(User.SAUL);
}
```

**Comment by FourElements** [comment]
Hello @letsky,

Did the same exact solution as you proposed when I was now testing it out. To be honest full novice in reactor, but I also believe the second approach is better, even if just by being less verbose :)

**Comment by simonbasle** [comment]
fixed by 33f418b
